src/envo/envo-edit.owl

```diff
@@ -5390,19 +5406,19 @@ SubClassOf(<http://purl.obolibrary.org/obo/ENVO_00000050> <http://purl.obolibrar
 
 # Class: <http://purl.obolibrary.org/obo/ENVO_00000051> (hot spring)
 
-AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "https://en.wikipedia.org/wiki/Hot_spring"^^xsd:string) <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_00000051> "A spring that is produced by the emergence of geothermally-heated groundwater from the Earth's crust."^^xsd:string)
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "https://en.wikipedia.org/wiki/Hot_spring"^^xsd:string) <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_00000051> "A spring through which groundwater, heated by geothermal energy, flows."@en)
```

Comment:
why complicate with 'groundwater' and 'heated by geothermal energy'.  simplify to
'a spring that emits water that is warmer than the ambient ground temperature'.   
Is 'warm spring' a concept in ENVO.  Generally 'hot' connotes a temperature above something like human body temperature.